### ISSUE TITLE
Unable to open old wallet due to an error when calling getHistory() on the bch wallet

### ISSUE THREAD

**Issue by kreitlowbryan** [blockchain/My-Wallet-V3-Frontend#1409]
I have never enabled the BCH wallet through the settings interface but the wallet address has some BTC in it that existed before the fork.

Here is the error seen in my developers console for Chrome Version 63.0.3239.84

<img width="913" alt="screen shot 2017-12-18 at 3 29 45 pm" src="https://user-images.githubusercontent.com/25312120/34133210-5fadf298-e408-11e7-99b0-53e617bc8e29.png">

This results in the Login button being greyed out but the Frontend Webapp being in a crashed state. Perhaps the code should be updated to check if that wallet exists or has been initialized?

Does anyone know if there is a way to initialize the wallet in the console so that I can access the webapp wallet?

https://github.com/blockchain/My-Wallet-V3-Frontend/blob/cf83bccef5411ee0565bd939514752c54db6239d/assets/js/services/wallet.service.js#L300

**Comment by plondon** [comment]
Hey @kreitlowbryan we're investigating a fix will keep you posted.

**Comment by plondon** [comment]
https://github.com/blockchain/My-Wallet-V3-Frontend/pull/1411

**Comment by alphabj** [comment]
I have a similar issue, except mine occurs at updateBchTxActivities. It immediately logs me out after logging in on any browser. I don't believe I've been accessing this wallet regularly since 2015, so something got messed up between then and now. Any ideas on how I can access it? Would prefer to discuss details over email: brad@bradyinger.com. Sorry if this is the wrong place but y'all have made yourselves hard to contact. Thanks.
<img width="553" alt="screen shot 2017-12-26 at 7 35 44 pm" src="https://user-images.githubusercontent.com/7626/34367152-de5808e6-ea74-11e7-920d-bc183c624ad0.png">

**Comment by plondon** [comment]
Hi @alphabj, let's check to see if v1.30.6 fixes your issue as well. If not I will investigate further. In terms of other ways to access this wallet, if you have the recovery phrase you could try generating a new wallet https://blockchain.info/wallet/#/recover and see if that resolves the issue.

**Comment by alphabj** [comment]
Thanks for the response @plondon, I was able to log in from the iOS app. Then I tried the upgrade option, but it just got stuck creating a new wallet and never finished. Then it would never let me back in again without trying to upgrade, even if I reinstalled the app. So I installed on my iPad. When I go to look at the recovery phrase, it just shows blank for each word. So I guess I’ll have to wait until I can use the webclient. I will also need to collect the bitcoin cash wallet, which only seems to be available from the web client as well. I have to pay a steep fee to transfer out of wallet right now anyways.
